src/transformers/models/vivit/modeling_vivit.py

```diff
@@ -547,8 +556,8 @@ def forward(
         >>> container = av.open(file_path)
 
         >>> # sample 32 frames
-        >>> indices = sample_frame_indices(clip_len=32, frame_sample_rate=1, seg_len=len(videoreader))
-        >>> video = videoreader.get_batch(indices).asnumpy()
+        >>> indices = sample_frame_indices(clip_len=32, frame_sample_rate=2, seg_len=2)
```

Comment:
This also isn't correct. `seq_len` should be the total number of frames in the video segment